static/js/tools/timeline/chart.tsx

```diff
@@ -190,6 +194,16 @@ class Chart extends Component<ChartPropsType> {
     this.loadDataAndDrawChart();
   }
 
+  private handleDenomInput(evt) {
+    const statVar = evt.target.value; // this is the search text
+    if (this.delayTimer) {
```

Comment:
Maybe no delayTimer now that this is called in onBlur. As a user, if I click outside the input box, I would expect the value to be submitted right away.

Reply:
Removed